### PR TITLE
UI Detail and xml

### PR DESCRIPTION
**Pull request by jsvivien**

### CODE REVIEW

**Comment by jsvivien** [comment]
ready

**Comment by jsvivien** [comment]
ready

**Comment by jsvivien** [comment]
ready

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -22,5 +22,12 @@
         <activity
             android:name=".screen.search.SearchActivity"
             android:theme="@style/AppThemeNoActionBar" />
+        <activity
+            android:name=".screen.detail_tv.DetailActivity"
+            android:screenOrientation="portrait"></activity>
```
Comment (DoanVanToan) [inline app/src/main/AndroidManifest.xml]: self close tag

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -22,5 +22,12 @@
...
+        <activity
+            android:name=".screen.trailer.TrailerActivity"
+            android:screenOrientation="landscape"></activity>
```
Comment (DoanVanToan) [inline app/src/main/AndroidManifest.xml]: self close tag

**Pull request merged**